docs/src/main/asciidoc/resteasy.adoc

```diff
@@ -615,7 +615,7 @@ This configuration option would recognize strings in this format (shown as a reg
 
 Once GZip support has been enabled you can use it on an endpoint by adding the `@org.jboss.resteasy.annotations.GZIP` annotation to your endpoint method.
 
-If you want to compress everything then we recommended that you use the `quarkus.http.enable-compression=true` setting instead to globally enable
+NOTE: TODO:should we just remove this? If you want to compress everything then we recommended that you use the `quarkus.http.enable-compression=true` setting instead to globally enable
```

Comment:
?

Reply:
It depends on the output of https://github.com/quarkusio/quarkus/pull/24558#discussion_r836169787 ;-)

Reply:
I've removed the TODO and changed the wording so that the config property is just ignored.